### PR TITLE
Remove unused templates

### PR DESCRIPTION
**Pull request by mbj**

### CODE REVIEW

**Comment by mbj** [comment]
@dkubb This is ready for review.

File: core/app/helpers/spree/base_helper.rb [hunk 1]
```diff
@@ -11,17 +11,6 @@ def body_class
       @body_class
     end
 
-    def flash_messages(opts = {})
```
Comment (dkubb) [inline core/app/helpers/spree/base_helper.rb]: @mbj FYI we do actually use this `flash_messages` method in two places in mrh-spree.

Comment (mbj) [inline core/app/helpers/spree/base_helper.rb]: @dkubb I intend to have the PR to move this code over ready before merging this. Removing the `merge-read` tag for now.

Comment (mbj) [inline core/app/helpers/spree/base_helper.rb]: I removed the removal of `flash_messages` for now.

**Comment by mbj** [comment]
@dkubb This is ready for the next pass (assuming we'll update the build dependencies on BK first).

**Pull request merged**